changes/ChangeSet-fix_auth-39979c2-2025.12.16-22.02.md

```
@@ -0,0 +1,121 @@
+| `scripts/README.md` |
+
+---
+
+## Statistics
+
+| Metric | Value |
+|--------|-------|
+| Total files changed |  |
+| New files | 2 |
+| Modified files | 1 |
+| Deleted files | 0
```

markdownlint-cli2 (0.18.1)

Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)